### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by MMGGYY66**
Adding a contact form to my portofolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,222 +1,267 @@
...
+        <section class="contact flex" id="contact">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </h3>
+            <form action="https://formspree.io/f/mayvdjkb" method="POST" class="form flex" id="contact-form">
+                <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" />
+                <div class="full-name flex">
+                    <input type="text" name="first_name" placeholder="First Name" maxlength="30" />
+                    <input type="text" name="last_name" placeholder="Last Name" maxlength="30" />
```
Comment (zieeco) [inline index.html]: - Hey, please, let's make this input field required to ensure the form is not submitted on an empty input field.

File: index.html [hunk 2]
```diff
@@ -1,222 +1,267 @@
...
+                </ul>
+            </div>
+        </section> <!-- end about me section -->
+
+        <!-- start contact form section -->
+        <section class="contact flex" id="contact">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </h3>
```
Comment (zieeco) [inline index.html]: - Hey, the text content here should be a paragraph and not a header. let current this to improve our markup

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @MMGGYY66 & @hansderly ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @MMGGYY66 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**